state-chain/pallets/cf-validator/src/delegation.rs

```diff
@@ -2,6 +2,14 @@ use codec::{Decode, Encode, MaxEncodedLen};
 use scale_info::TypeInfo;
 use serde::{Deserialize, Serialize};
 
+/// The minimum operator fee we charge.
```

Comment:
nit: "we" don't charge it. Also could be useful to clarify that it's in bps.


```suggestion
/// The minimum delegation fee that can be charged, in basis points.
```